### ISSUE TITLE
[Bug] Discord is spamming Sentry error popups, making the app practically unusable

### ISSUE THREAD

**Issue by ColinTimBarndt** [Vendicated/Vencord#2698]
### What happens when the bug or crash occurs?

Whenever I do anything, this popup is being opened over and over again:
![image](https://github.com/user-attachments/assets/cd30381c-48db-4bba-82a5-f517c73758ba)
Also, you have to dismiss every popup before the app is accepting any further interactions.

### What is the expected behaviour?

These popups should obviously not be there. I suppose Discord changed something about their Sentry packages.

### How do you recreate this bug or crash?

1. Open [Discord Version 0.0.60](https://dl.discordapp.net/apps/linux/0.0.60/discord-0.0.60.deb).

### Errors

![image](https://github.com/user-attachments/assets/cd30381c-48db-4bba-82a5-f517c73758ba)

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well
- [X] I have read the requirements for opening an issue above

**Comment by Sqaaakoi** [comment]
Are you sure this is a Vencord issue? Does this only occur after injecting Vencord into the stock client? I didn't quickly find any **native code** relating to Sentry

**Comment by ColinTimBarndt** [comment]
Ah sorry, I should've checked. Let's hope that Discord fixes this soon.

**Comment by ColinTimBarndt** [comment]
For those after me who get here in search if a solution:

Here's the fix (Ubuntu/pop!os) by downgrading back to 0.0.59:
1. Download https://dl.discordapp.net/apps/linux/0.0.59/discord-0.0.59.deb
2. Install it: `sudo dpkg -i ~/Downloads/discord-0.0.59.deb`
3. Disable immediate update: `open ~/.config/discord/settings.json` and add one line at the bottom:
```json
{
  // ...
  "MIN_HEIGHT": 500,
  "chromiumSwitches": {}, // <- don't forget the ,
  "SKIP_HOST_UPDATE": true // <- this line
}
```
4. Optionally repair Vencord: `sh -c "$(curl -sS https://raw.githubusercontent.com/Vendicated/VencordInstaller/main/install.sh)"`
5. Start Discord

**Comment by alex73630** [comment]
I'd like to re-open this issue as this only happens *after* injecting Vencord into the stock client with a version above 0.0.59.

The error doesn't show when Vencord is not injected and can be reproduced 100% of the time.

I wonder if it could be linked to the NoTrack plugin in some way

**Comment by spike11302000** [comment]
This is still a issues for me on 0.0.69, Debian 12

**Comment by ColinTimBarndt** [comment]
@Vendicated I'd consider looking into this issue again. I can't confirm the issues at the moment as I'm not using Vencord anymore for now.

**Comment by alex73630** [comment]
I wanted to share that I've been running this fix ( #2706 ) for a couple of weeks now and while I understand it's really hacky, it does the job pretty well.

I've been able to keep up with updates with no issues so far.

**Comment by lewisakura** [comment]
Thank you for letting us know. We'll try and work on a fix for this, but since it's not an issue that can be easily reproduced consistently (for instance, I have never had it happen at all) it may take some time. If there are *specific* reproduction steps that anyone has to consistently reproduce the issue from a clean install, please send them here.

**Comment by Dwyriel** [comment]
Been having this issue since discord version 0.0.60 and newer and could easily replicate it on Arch Linux. Tried in 3 different machines (clean discord installs or not), and in all 3 of them, vencord would only work properly on discord version 0.0.59 or below (installed from the arch repo). Anything above that led to error.

**Comment by spike11302000** [comment]
So i think found a way to fix the issue without #2706. completely apt remove discord then reinstall then install vencord, havent had the popup yet. im think there maybe files that dont get remove/updated when updating to newer versions but i yet to confirm this.

**Comment by skajmer** [comment]
> So i think found a way to fix the issue without #2706. completely apt remove discord then reinstall then install vencord, havent had the popup yet. im think there maybe files that dont get remove/updated when updating to newer versions but i yet to confirm this.

that actually worked for me

**Comment by ajayyy** [comment]
Removing discord and deleting `/opt/discord` worked for me

**Comment by Vendicated** [comment]
I tried testing this in an ubuntu VM back then but couldn't reproduce. The error also comes from Discord's sentry library which Vencord doesn't interact with in any way, so it doesn't make sense for this to happen

It would be useful if anyone affected could debug it further and share their findings / a patch

**Comment by alex73630** [comment]
I can also confirm that removing the `discord` package on Arch (or whatever distro you are using) and deleting the `/opt/discord` folder did fix the issue.

I suspect there is a leftover file somewhere that is causing this issue and I sadly didn't take the time to backup the folder before deleting it.

Also the fix from #2706 now causes crashes on more recent builds of the Discord package so I now recommend against using it.

TLDR: Uninstall Discord, removing the `/opt/discord` folder and reinstall Discord fixes the issue.

**Comment by spike11302000** [comment]
> I tried testing this in an ubuntu VM back then but couldn't reproduce. The error also comes from Discord's sentry library which Vencord doesn't interact with in any way, so it doesn't make sense for this to happen
> 
> It would be useful if anyone affected could debug it further and share their findings / a patch

I tried for quite a bit and was unable to replicated the issues. my best guess is some files didnt get properly removed between updates and vencord was causing a weird interaction with them causing the issue.
